test/tutorial/step-3-setting-up-supertest.md

```diff
@@ -0,0 +1,19 @@
+### Setting up Supertest
+
+First we need to install supertest:
+
+```
+npm install supertest --save-dev
+```
+
+To use supertest we need to require supertest and app into your test file, to do this add this to the top of your `server.test.js` file:
+
+```
+const supertest = require('supertest')
+const app = require('../server')
+const request = supertest(app)
+```
+
+The `request` is what will be going to your server, we will be asserting on the response that comes back.
```

Comment:
I'd reword "asserting" here, might not mean much to a beginner

Reply:
Nice pick up, it's interesting how automatic this language becomes! I have changed it so it is less jargon-y.